sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/jproxy/ConnectionProperties.java

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.messaging.eventhubs.jproxy;
```

Comment:
Why is this test package called `jproxy`?

Reply:
This is pulled from Track 1 where they had pulled in a simple jproxy written by one of the devs. I don't have any preferences if you want to name it something else.
https://github.com/SreeramGarlapati/jproxy